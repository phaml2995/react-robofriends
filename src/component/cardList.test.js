import { shallow } from 'enzyme';
import CardList from './cardList.component';
import React from 'react';

it('expect to render Card Comp', () => {
    const mock = [
        {
            id: 1,
            name: 'john snow',
            username: 'johnjohn',
            email: 'john@gmail.com'
        }
    ]
    expect(shallow(<CardList robots={mock} />)).toMatchSnapshot()
})