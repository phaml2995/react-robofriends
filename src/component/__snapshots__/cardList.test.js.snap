// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`expect to render Card Comp 1`] = `ShallowWrapper {}`;
